### DISCUSSION TITLE
Fauxmo Hue devices not supported for Alexa groups?

### DISCUSSION THREAD

**Discussion by Miq1** [vintlabs/fauxmoESP#140] [General]
Yet another surprise... 😞 

I tried to add devices with the Fauxmo Hue emulation to a group in Alexa. That is failing with a message "The device does not support the requested value" (translated from German "Gerät unterstützt den angeforderten Wert nicht") and the device is not added to the group.

Any workaround for that?

So far the move from the former Belkin emulation to Hue does not convince me really...

**Comment by pvint** [comment]
I normally use groups heavily for my Fauxmo devices, but since a recent move I haven't... I just tested, and it seems to work fine for me.

I'm curious, did you "discover" the devices using an Echo device or through the Alexa App on a phone?

**Reply by Miq1** [reply]
I am doing the discovery in the app. Would that make a difference? 
I will try today to setup routines to switch more devices in a row. If that will work, I at least have a workaround. Wifey does not likey... 😉

**Reply by Miq1** [reply]
The workaround with a routine does work. Instead of grouping the Fauxmo devices I added an explicit switch command for each device to a routine named like the former group switch command. This is a bit more configuration effort, but at least it is functional.

**Reply by Miq1** [reply]
I also tried to delete a group and set it up again to no avail. There was a strange effect with that, as a device not explicitly mentioned by me was automatically added to the group and immediately got the error message mentioned. Even taking that device out of the group was not successful, with the next addition of another it popped up again.

**Reply by Miq1** [reply]
Final experiment was to have a Fauxmo device added through the Echo dot instead of the app. Makes no difference at all.

**Reply by Shashank-Shastri** [reply]
@Miq1 I had the same issue. I solved it by just making Alexa add the devices to groups instead of me adding on app/website.

**Reply by Miq1** [reply]
> 
> 
> @Miq1 I had the same issue. I solved it by just making Alexa add the devices to groups instead of me adding on app/website.

How did you tell Alexa to do that?

**Reply by Shashank-Shastri** [reply]
Just say "Alexa add `device name` to `group name`". You'll mostly get "I could not find that device, did you mean `your device name`?", just say yes.

**Reply by Shashank-Shastri** [reply]
Though I still see that "The device does not support the requested value" on app. I can still control the device, but would be great if there's a solution for this.
